src/gamescope/manager.rs

```diff
@@ -45,6 +46,18 @@ impl Manager {
         }
     }
 
+    /// Starts the wayland manager and adds its dbus interface
+    pub async fn start_wayland_manager(&self) -> Result<(), Box<dyn Error>> {
```

Comment:
It's possible that Gamescope is not running when gamescope-dbus is started. We should be able to handle Gamescope starting/stopping, etc. Instead of creating this interface right away, we should instead use `watcher` to monitor the `XDG_RUNTIME_DIR` with inotify for any new `gamescope-*` socket files and create the interface in response to that. Similar to how we do this with `watch_xwaylands`.